### PR TITLE
Added 2 features.

### PR DESCRIPTION
**Pull request by EssamEmad**
## Dynamic Progress

I added a feature where we can make the progress bar dynamic. That is being able to change the progress at anytime during runtime. Specifically in the following lines of code in the onDraw method and the updateView method which is responsible for updating the progress UI:

``` java
 if(resetToZero) {
            width = 0;
            resetToZero = false;
            invalidate();
        } else if(width < maxWidth) {
            width+= this.speed;
            invalidate();
        }
```

``` java
 /**
     * call the function when you want to update view
     */
    public void updateView() {
        resetToZero = true;
        initView();
        invalidate();
    }
```
## Changing Speed

Another feature I added is to be able to change the speed of the animation that the progress bar is moving with. It is done by adding a private variable speed and incrementing the width by speed. The speed can be set using this method call:

``` java
 public void setSpeed(int speed){
        this.speed = speed;
    }
```

### CODE REVIEW

File: StaticProgressBar/app/src/main/java/nayoso/staticprogressbar/CustomProgress.java [hunk 1]
```diff
@@ -45,24 +39,26 @@ of this software and associated documentation files (the "Software"), to deal
...
     private TextView textView;
-	private int width = 0;
+    private int width = 0;
     private int maxWidth = 0;
     private int maxHeight = 0;
     private int progressColor;
     private int progressBackgroundColor;
     private int progressShape = SHAPE_RECTANGLE;
-    private float maximumPercentage = 0.0f;
+    private float maximumPercentage = .0f;
```
Comment (NikoYuwono) [inline StaticProgressBar/app/src/main/java/nayoso/staticprogressbar/CustomProgress.java]: Any special reason why do you change it to .0f?

Comment (EssamEmad) [inline StaticProgressBar/app/src/main/java/nayoso/staticprogressbar/CustomProgress.java]: No, I must have changed it by mistake!

Comment (NikoYuwono) [inline StaticProgressBar/app/src/main/java/nayoso/staticprogressbar/CustomProgress.java]: OK, beside that LGTM! Thank you btw

Comment (EssamEmad) [inline StaticProgressBar/app/src/main/java/nayoso/staticprogressbar/CustomProgress.java]: Cool, no problem :)

**Pull request merged**